src/pysdmx/util/handlers.py

```diff
@@ -0,0 +1,59 @@
+"""Handlers file provide functions to make the code more readable."""
+
+from typing import Any
+
+
+# import pandas as pd
+
+
+def split_unique_id(obj_: str) -> tuple[str, str, str]:
+    """Split unique id.
+
+    Args:
+        obj_: String with the name of the object.
+
+    Returns:
+        agencyID, id and version.
+    """
+    data = obj_.split(":", 1)
+    agencyID = data[0]
+    data = data[1].split("(", 1)
+    id = data[0]
+    version = data[1].split(")", 1)[0]
+
+    return agencyID, id, version
+
+
+def split_from_urn(obj_: str, split_id: bool = True) -> Any:
+    """Split from urn.
+
+    Args:
+        obj_: String with the name of the object.
+        split_id: True splits the id.
+
+    Returns:
+        full id.
+    """
+    full_id = obj_.split("=", 1)[1]
+    return split_unique_id(full_id)
+
+
+def add_list(element: Any) -> Any:
+    """Adds data contained in the xml Dataset into a list if it is possible.
```

Comment:
I guess this is not xml specific, isn't it (considering its location in the package)? If so, I'd remove xml from the docstring. It's also hard for me to understand what this is about and why it should be used. Maybe some more documentation could be added? Else, if this is io-specific only, we could also have a util module below the io module. As you wish :)

Reply:
Yes, this is due to how xmltodict processes the XML, if there is only one child element, it creates a dictionary, but if there is more than one child element, it creates a list of dictionaries. 

This is io specific only, and specific only for xml, so I will move it to io.xml.utils.py and update the docstring

Reply:
Solved in latest commit.